src/node.cc

```diff
@@ -981,6 +983,42 @@ void SetupNextTick(const FunctionCallbackInfo<Value>& args) {
       FIXED_ONE_BYTE_STRING(args.GetIsolate(), "_setupNextTick"));
 }
 
+void PromiseRejectCallback(PromiseRejectMessage message) {
+  Isolate* isolate = Isolate::GetCurrent();
```

Comment:
`Isolate::GetCurrent()` is going to be deprecated. Another way of retrieving the isolate should be used.